src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -51,6 +52,9 @@
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Data" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'net45'">
+    <Compile Include="System\Data\Common\DbDataReaderExtensions.Facade.cs" />
```

Comment:
Typically I believe we keep the Compile Includes separate. But since there is just one file and this partial facade should not grow in size ideally, I will combine the two